azkaban-exec-server/build.gradle

```diff
@@ -5,11 +5,13 @@ dependencies {
   compile('org.apache.kafka:kafka-log4j-appender:0.10.0.0')
   compile('com.googlecode.json-simple:json-simple:1.1.1')
 
+  runtime(project(':azkaban-hadoop-security-plugin'))
+  runtime('org.slf4j:slf4j-log4j12:1.7.18')
```

Comment:
There were no changes to dependencies. These lines were just moved from below to top